scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,218 @@
+
+    is_installer_sdk = manifest.is_sdk_engine(engine_path=ctx.engine_path)
+
+    def run_android_command(args, cwd, error_msg):
+        has_errors = exp.process_command(args, cwd=cwd)
+        if has_errors:
+            raise exp.ExportProjectError(error_msg)
+    
+    run_android_command([o3de_cli_script_path, 'android-configure', '--validate'], cwd=ctx.project_path, error_msg="Invalid Android Configuration")
+
+    # Calculate the tools and game build paths
+    default_base_path = ctx.engine_path if engine_centric else ctx.project_path
+
+    # Resolve (if possible) and validate any provided seedlist files
+    validated_seedslist_paths = exp.validate_project_artifact_paths(project_path=ctx.project_path,
+                                                                    artifact_paths=seedlist_paths)
+    
+    # Preprocess the seed file paths in case there are any wildcard patterns to use
+    preprocessed_seedfile_paths = exp.preprocess_seed_path_list(project_path=ctx.project_path,
+                                                                paths=seedfile_paths)
+    
+    eutil.process_level_names(ctx, preprocessed_seedfile_paths, level_names, ASSET_PLATFORM)
+
+    tools_build_path = eutil.handle_tools(ctx, should_build_tools, is_installer_sdk, tool_config, '', tools_build_path, default_base_path, engine_centric, logger)
+
+    eutil.handle_assets(ctx, should_build_all_assets, tools_build_path, is_installer_sdk, 
+                        tool_config, engine_centric, fail_on_asset_errors,
+                        ASSET_PLATFORM, ASSET_PLATFORM, validated_seedslist_paths, preprocessed_seedfile_paths, 
+                        ctx.project_path/'AssetBundling', max_bundle_size,
+                        logger)
+
+    android_project_config= android_support.get_android_config(project_path=None)
+
+    android_sdk_home = pathlib.Path(android_project_config.get_value(key=android_support.SETTINGS_SDK_ROOT.key))
+
+    run_android_command([o3de_cli_script_path, 'android-generate', '-p', ctx.project_name, '-B', target_android_project_path, "--asset-mode", asset_pack_mode],
+                        cwd=default_base_path,
+                        error_msg="Android Project Generation Failure.")
+
+    run_android_command([target_android_project_path / ('gradlew'+ android_support.GRADLE_EXTENSION), f'assemble{build_config.title()}'],
+                        cwd=target_android_project_path,
+                        error_msg="Gradle Build Failure.")
+    
+    if deploy_to_device:
+        if not activity_name:
+            activity_name = f'{ctx.project_name}Activity'
+
+        if not org_name:
+            org_name = f'org.o3de.{ctx.project_name}'
+        android_support.deloy_to_android_device(target_android_project_path,
+                                build_config,
+                                android_sdk_home,
+                                org_name,
+                                activity_name)
```

Comment:
Can we line up these variables (not just here, but also everywhere, including the function declarations). Do you have tabs as space set properly?